template_config.yml

```diff
@@ -29,16 +35,22 @@ plugin_dash_short: python
 plugin_default_branch: master
 plugin_name: pulp_python
 plugin_snake: pulp_python
+post_job_template: null
+pre_job_template: null
 publish_docs_to_pulpprojectdotorg: true
+pulp_scheme: https
 pulp_settings: null
 pulpcore_branch: 3.13
 pulpcore_pip_version_specifier: ~=3.13.0
 pulpprojectdotorg_key_id: null
 pydocstyle: true
 pypi_username: pulp
+python_version: '3.6'
```

Comment:
when using 3.6, you should use pulp_scheme: HTTP
we don't have https for 3.6 image